qiita_db/portal.py

```diff
@@ -0,0 +1,53 @@
+from .sql_connection import SQLConnectionHandler
+from .util import convert_to_id
+from qiita_core.qiita_settings import qiita_config
+
+# Only make functions available for main portal
+if qiita_config.portal == "QIITA":
+    def get_studies_by_portal(portal):
```

Comment:
As we previously discussed, these are essentially helper functions for the `qiita-env` command line commands so to me it makes no sense to put them into an object.